src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java

```diff
@@ -0,0 +1,69 @@
+package io.github.bobocodebreskul.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.github.bobocodebreskul.context.annotations.Get;
+import io.github.bobocodebreskul.context.registry.BringContainer;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DispatcherServlet extends HttpServlet {
+
+  private final BringContainer container;
+
+  private final Map<String, Object> pathToController;
+
+  public DispatcherServlet(BringContainer container, Map<String, Object> pathToController) {
+    this.container = container;
+    this.pathToController = pathToController;
+  }
+
+  private static Optional<Method> getGetMethod(Object bean) {
+    return Arrays.stream(bean.getClass().getMethods())
+        .filter(m -> m.getAnnotation(Get.class) != null).findFirst();
+  }
+
+  @SneakyThrows
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+      throws ServletException, IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    Object bean = pathToController.get(req.getPathInfo());
+    Optional<Method> method = getGetMethod(bean);
+    Object result;
+    if (method.isPresent()) {
+      result = method.get().invoke(bean);
+      resp.setStatus(200);
+      PrintWriter writer = resp.getWriter();
```

Comment:
This piece of code could be extracted to a separate method to avoid code duplication
```
      PrintWriter writer = resp.getWriter();
      writer.println(mapper.writeValueAsString(result));
      writer.flush();
 ```

Reply:
it will be done in next PR